js/analyzer.js

```diff
@@ -308,32 +255,8 @@ YoastSEO.Analyzer.prototype.stopwords = function() {
  * @returns {result object}
  */
 YoastSEO.Analyzer.prototype.fleschReading = function() {
-	if ( this.preProcessor.__store.wordcountNoTags > 0 ) {
-		var score = (
-			206.835 -
-				(
-					1.015 *
-						(
-							this.preProcessor.__store.wordcountNoDigits /
-							this.preProcessor.__store.sentenceCountNoTags
-						)
-					) -
-						(
-							84.6 *
-						(
-					this.preProcessor.__store.syllablecount /
-					this.preProcessor.__store.wordcountNoDigits
-				)
-			)
-		)
-		.toFixed( 1 );
-		if ( score < 0 ) {
-			score = 0;
-		} else if ( score > 100 ) {
-			score = 100;
-		}
-		return [ { test: "fleschReading", result: score } ];
-	}
+	var calculateFleschReading = require( "./analyses/calculateFleschReading.js" );
+	return [ { test: "fleschReading", result: calculateFleschReading( this.config.text ) } ];
```

Comment:
The checks for the score being higher than 100 and lower than 0 should still be here.